cmd/recorder/recorder.go

```diff
@@ -138,13 +149,21 @@ func (rec *Recorder) runBrowser(recURL string) error {
 
 	log.Printf("stop received, shutting down browser")
 
+	var disconnected bool
+	disconnectCheckExpr := "window.callsClient.disconnect(); !window.callsClient || window.callsClient.closed"
 	if err := chromedp.Run(ctx,
-		chromedp.EvaluateAsDevTools("window.callsClient.disconnect();", nil),
+		chromedp.Evaluate(disconnectCheckExpr, &disconnected),
```

Comment:
Just to make sure I understand, this evals the `disconnectCheckExpr`, and waits for the eval to finish (so it must be a blocking call, but without a timeout), and then if it returns false, we... do the same thing we do if it was successful?

Reply:
Yes, it's really just for logging/debugging purposes as we are shutting down the process anyway but would prefer to do it cleanly if possible.

Reply:
Ah, ok, makes sense.